src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
```

Comment:
According to the checklist, we shouldn't use static methods or static fields unless it's constants. In this case, `COLOR_SUPPLIER` and `RANDOM` should be instance fields rather than static fields. Consider creating them as non-static fields and initializing them in a constructor or directly in the field declaration.